frontend/src/js/controllers/profileCtrl.js

```diff
@@ -123,6 +123,43 @@
             });
         };
 
+        // Disable User Account
+        vm.confirmdisableUserAccount = function(ev) {
+            ev.stopPropagation();
+            // Confirms that the user wants to deactivate the account
+            var confirm = $mdDialog.confirm()
+                .title('Would you like to deactivate your account?')
+                .textContent('Note: This action will deactivate your account and you will not be able to login again.')
```

Comment:
@Suryansh5545 Can you check if you can add a page like this instead: https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/views/web/challenge/cancel-submission.html?

Check the code in challenge controller here: https://github.com/Cloud-CV/EvalAI/blob/3af33a2aee2f7a6dab90ed574dd3e2178d0901e8/frontend/src/js/controllers/challengeCtrl.js#L2079-L2114